automl/google/cloud/automl_v1beta1/proto/image_pb2.py

```diff
@@ -540,6 +576,14 @@
           xportModel]) and used on a mobile device with Core ML
           afterwards. Expected to have a higher latency, but should also
           have a higher prediction quality than other models.
+      node_qps:
+          Output only. An approximate number of online prediction QPS
+          that can be supported by this model per each node on which it
+          is deployed.
+      node_count:
+          Output only. The number of nodes this model is deployed on. A
+          node is an abstraction of a machine resource, which can handle
+          online prediction QPS as given in the node\_qps field.
   """,
         # @@protoc_insertion_point(class_scope:google.cloud.automl.v1beta1.ImageClassificationModelMetadata)
```

Comment:
Add `node_qps` and `node_count` to `ImageClassificationModelMetadata` in `v1beta1`